controllers/tobiko_controller.go

```diff
@@ -71,12 +71,33 @@ func (r *TobikoReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		return ctrl.Result{}, err
 	}
 
+	logging := log.FromContext(ctx)
+	if r.CompletedJobExists(ctx, instance) {
+		// The job created by the instance was completed. Release the lock
+		// so that other instances can spawn a job.
+		logging.Info("Job completed")
+		r.ReleaseLock(ctx, instance)
+	}
+
+	operatorLabels := map[string]string{
+		"operator":  "test-operator",
+		"test-name": instance.Name,
+	}
+
+	serviceLabels := map[string]string{
+		common.AppSelector: tobiko.ServiceName,
+
 	// Check whether the user wants to execute workflow
 	workflowActive := false
 	if len(instance.Spec.Workflow) > 0 {
 		workflowActive = true
 	}
 
+	// Combine labels
+	for key, value := range operatorLabels {
+		serviceLabels[key] = value
+	}
+
```

Comment:
Do we need to combine the `operatorLabels` and `serviceLabels` like this? Can't we do something like:
```
serviceLabels := map[string]string {
  "operator": "test-operator",
  "test-name": instance.Name,
  common.AppSelector: tobiko.ServiceName
}
```